jackson-apt-processor/src/main/java/org/dominokit/jacksonapt/processor/Type.java

```diff
@@ -24,8 +24,11 @@
 import javax.lang.model.type.DeclaredType;
 import javax.lang.model.type.TypeKind;
 import javax.lang.model.type.TypeMirror;
+import javax.lang.model.util.SimpleTypeVisitor6;
```

Comment:
I think class is deprecated in java9 ..should we use SimpleTypeVisitor8 instead?

Reply:
Good suggestion. I will do it.

Reply:
Done!